packages/tables/src/Concerns/HasFilters.php

```diff
@@ -221,7 +221,15 @@ public function parseTableFilterName(string $name): string
 
     public function getTableFiltersSessionKey(): string
     {
-        $table = md5($this::class);
+        $tenantKey = filament()->getTenant()?->getKey();
```

Comment:
`filament/filament` might not be installed when using `filament/tables`, so there needs to be some `class_exists()` check probably on the `Filament` facade. Then replace `filament()->` with `Filament::`